res/controllers/common-controller-scripts.js

```diff
@@ -368,6 +368,7 @@ bpm.tapButton = function(deck) {
 // ----------------- Common regular expressions --------------------------
 script.samplerRegEx = /\[Sampler(\d+)\]/ ;
 script.channelRegEx = /\[Channel(\d+)\]/ ;
+script.channelRegExExact = /^\[Channel(\d+)\]$/ ;
```

Comment:
> Hm, it seems a shame to have to duplicate the regexes just to add ^$.

Yeah kinda, but I don't know if other regexes would benefit from such a helper function (which is probably why there isn't a standard JS helper function for it).

> Isn't that what eqRegEx or quickEffectRegEx would give you? (i.e. group.match(eqRegEx)[1] would give you the eq/quickeffect rack's group).

I mean there may be a use case for pulling `[Channel1]` out of `[EqualizerRack1_[Channel1]_Effect1]`. eqRegEx and quickEffectRegEx would not do that.

> (also maybe we should remove the square brackets from eqRegEx / quickEffectRegEx so that the user doesn't have to re-add them to the matched value to get the group)

I don't understand what you mean.